sonar-db/src/main/java/org/sonar/db/user/UserDao.java

```diff
@@ -177,6 +179,7 @@ public boolean deactivateUserByLogin(DbSession dbSession, String login) {
     mapper.deleteUserMeasureFilterFavourites(dto.getId());
     mapper.deleteUserProperties(dto.getId());
     mapper.deleteUserRoles(dto.getId());
+    mapper.deletePropertiesMatchingLogin(asList(DEFAULT_ISSUE_ASSIGNEE), dto.getLogin());
```

Comment:
to deactivate a user, either we should use a specific mapper/dao or we should have a service that call the different dao. That's how we've done at least. I don't ask you to change that now, but I want us to agree on what should be done